panels/user-accounts/um-password-dialog.c

```diff
@@ -115,13 +141,25 @@ cancel_password_dialog (GtkButton        *button,
                         UmPasswordDialog *um)
 {
         finish_password_change (um);
+        shell_enable_password_mode (um, FALSE);
```

Comment:
No need to call this here, since `finish_password_change()` already turns it off